src/components/markdown-htmlview.component.js

```diff
@@ -266,19 +306,33 @@ export class MarkdownHtmlView extends Component {
             styleText.textAlign = cellAlign[attribs.style];
           }
 
+          const Component =
+            node.children.filter(
+              elem => elem.type === 'tag' && elem.name === 'img'
```

Comment:
Do you think it would be possible to move this arrow function in a separate function, if it is used more than once?

Reply:
done!